src/PlatformCompatibilityTests/ServiceControlCompatibilityTests/SampleTests.cs

```diff
@@ -1,26 +1,50 @@
 ﻿namespace ServiceControlCompatibilityTests
 {
     using NUnit.Framework;
+    using System;
+    using System.Linq;
 
     [TestFixture]
     class SampleTests : SqlScTest
     {
         [Test]
-        public void SampleTestA()
+        public async void SampleTestA()
         {
-            var details = ServiceControl.Get<ServiceControlDetails>("");
```

Comment:
Rather than exposing "Get" method and having DTOs all over the place, think we''d be better off if ServiceControl wrapper would only expose a few meaningful methods to work with rather than generic ones. Might be out of scope for this PR though?